.teamcity/settings.kts

```diff
@@ -1088,3 +1062,100 @@ object WpDesktop_DesktopE2ETests : BuildType({
 	}
 })
 
+object WPComPlugins : Project({
+    name = "WPCom Plugins"
+    buildType(WPComPlugins_EditorToolKit)
+    params {
+        text("docker_image_wpcom", "registry.a8c.com/calypso/ci-wpcom:latest", label = "Docker image", description = "Docker image to use for the run", allowEmpty = true)
+    }
+})
+
+object WPComPlugins_EditorToolKit : BuildType({
+    name = "Editor ToolKit"
+
+    artifactRules = "editing-toolkit.zip"
+
+    vcs {
+        root(WpCalypso)
+        cleanCheckout = true
+    }
+
+    steps {
+        script {
+            name = "Prepare environment"
+            scriptContent = """
+                #!/bin/bash
+
+                # Update node
+                . "${'$'}NVM_DIR/nvm.sh" --no-use
+                nvm install
+
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+
+                # Update composer
+                composer install
+
+                # Install modules
+                yarn install
```

Comment:
Is there a way to share this step across wp-calypso builds?

Reply:
I don't think so :( Each build _instance_ always starts with a clean checkout. Even if there were a pre-populated `node_modules` (there is none at the moment), we still have to run `yarn install` in case that particular branch has introduced a new dependency.

`yarn install` takes about 2 minutes in many builds, it is definitely something I'm trying to improve. Potential solutions include migrating to Yarn Plug'n'Play, or have some kind of pre-populated node_modules in the base docker image.